tinygrad/tensor.py

```diff
@@ -2995,7 +2997,7 @@ def _broadcasted(self, y:Union[Tensor, UOp, ConstType], reverse:bool=False, matc
     return x._broadcast_to(out_shape:=_broadcast_shape(x.shape, y.shape)), y._broadcast_to(out_shape)
 
   def _to_const_val(self, x:Union[Tensor, ConstType]) -> Union[Tensor, ConstType]:
```

Comment:
This should be removed from Tensor

Reply:
yea I hate that tensor is accessing args to begin with, I'll make this an actual wrapper that asserts.